### DISCUSSION TITLE
Clicking active tab could open the previous active tab

### DISCUSSION THREAD

**Discussion by frandmb** [zen-browser/desktop#978] [Ideas]
Useful productivity feature to do quick looks. Idea taken from Sidebery

**Comment by SneezingCactus** [comment]
I second this! There are times where I have a lot of open tabs and I have to switch between two tabs that are in opposite sides of the tab list, or between two tabs in different workspaces, and it becomes a real hassle to have to search for that last tab. This is definitely a feature I'd love to see added.
